### ISSUE TITLE
The Guild 3 (311260)

### ISSUE THREAD

**Issue by Blisto91** [ValveSoftware/Proton#5317]
# Compatibility Report
- Name of the game with compatibility issues: The Guild 3
- Steam AppID of the game: 311260

## System Information
- GPU: AMD R9 380
- Driver/LLVM version: Mesa 21.2.5/13.0.0
- Kernel version: 5.15.2-arch1-1
- Link to full system information report as [Gist](https://gist.github.com/Blisto91/13671f10a0b7e4ebe52b5a5533037ce8): https://gist.github.com/Blisto91/13671f10a0b7e4ebe52b5a5533037ce8
- Proton version: 5.13-6, 6.3-7, 7.0-1

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
Game only starts when SafeMode is selected 

[steam-311260.log](https://github.com/ValveSoftware/Proton/files/7533538/steam-311260.log)

## Reproduction
Launch game -> select normal mode (not SafeMode) -> game doesn't start/shuts down shortly after start

## Other info
Launching the game in SafeMode the game gets you into the main menu and i can correctly start a game. Haven't tested much more yet.
Unsure if the issue is with the intro splashes.

Launching with Proton-6.21-GE-1 yields a similar result except here you get to hear sound from one of the intro splash screen before it either shuts down again like experimental or just freezes

Edit: added 7.0-1 since it is still a issue.

**Comment by Blisto91** [comment]
Okay yes it has to do with the intro videos. If i remove the files in (or where ever you have the game installed)
```
~/.local/share/Steam/steamapps/common/The Guild 3/media/Video/
```
then it launches correctly without SafeMode

Edit:
An alternative method is to disable the intro videos through the games config.
In `options.cfg` located at `steamapps/compatdata/311260/pfx/drive_c/users/steamuser/AppData/Local/Guild3/` add this: 
```
[Debug]
ShowIntroVideos = false
```
Note for this method any changes in the games settings will overwrite it and you will need to add it again.

**Comment by Blisto91** [comment]
In game overview map has graphical issues in 6.3-7 and Experimental. 5.13-6 looks fine.
I am guessing this is a DXVK issue.

<details>
  <summary>Proton 6.3-7 & Experimental</summary>
  
![2021-11-16_20-45](https://user-images.githubusercontent.com/47954800/142056255-d58e84dd-44af-4537-b3bf-e1e9281f22d6.png)

</details>
<details>
  <summary>Proton 5.13-6</summary>
  
![2021-11-16_20-49](https://user-images.githubusercontent.com/47954800/142056272-324a7845-e084-42e4-b0b0-ed3b2b7b4a7e.png)

</details>

**Comment by Blisto91** [comment]
Narrowed the overview map issue down to happening in DXVK 1.9.0

**Comment by alasky17** [comment]
> Narrowed the overview map issue down to happening in DXVK 1.9.0

@Blisto91 Thank you!  The best place to report that is directly to the DXVK folks: https://github.com/doitsujin/dxvk/issues

**Comment by Blisto91** [comment]
> @Blisto91 Thank you! The best place to report that is directly to the DXVK folks: https://github.com/doitsujin/dxvk/issues

Yes I know I just gotta get off my butt and collect all the logs. Try the apitrace stuff 😁 

Edit: [Upsteam issue](https://github.com/doitsujin/dxvk/issues/2380)

**Comment by Blisto91** [comment]
Overview map visual artifacts have now been fixed on DXVK master

Edit: Fix is also now included in proton experimental
